build/test-selenium-requirements.txt

```diff
@@ -1,5 +1,2 @@
 pytest
-jupyter_client==6.1.7
-notebook==6.4.10
-pyzmq==19.0.2
-selenium==3.141
```

Comment:
Oh, it is nice not having all these requirement constraints!

Reply:
they have all been consolidatated under src\Python\requirements.txt